facia-press/app/frontpress/FapiFrontPress.scala

```diff
@@ -393,15 +415,17 @@ trait FapiFrontPress extends EmailFrontPress with Logging {
     for {
       config <- Response.Async.Right(fapiClient.config)
       collectionIds = collectionsIdsFromConfigForPath(path, config)
-      pressedCollections <- Response.traverse(collectionIds.map(generateCollectionJsonFromFapiClient))
+      pressedCollections <- Response.traverse(collectionIds.drop(1).map(generateCollectionJsonFromFapiClient))
```

Comment:
what's the motivation for the drop(1) here?

Reply:
Ah good spot, thanks! That's definitely not supposed to be there still. I added it to work around a CAPI error I was seeing relating to the first collection of the test front I'm working with, but I think now's the time to get to the bottom of the error.

Reply:
Fixed in 49d9adb4d41287001716468814eb8a92a48582c0. For context it turns out the first collection was breaking things because of a weirdly configured backfill (in CODE). Fixed with Gareth's help and the whole front presses successfully now (without the `drop`).